### PR TITLE
add introduction and terminology for mobile ci/cd

### PR DESCRIPTION
**Pull request by philbrookes**
It looks like quite a lot of this information was already present. I have updated the description and added common terminology.

### CODE REVIEW

File: modules/ROOT/pages/_partials/generic-service-intros.adoc [hunk 1]
```diff
@@ -49,7 +49,9 @@ The {keycloak-service} service allows you to add authentication and authorizatio
 = {digger-service}
 //tag::mobilecicd-service[]
 
-The {digger-service} service allows you automate the building of your native and hybrid mobile app for both Android and iOS.
+The {digger-service} service provides a system to bring together the product of developers work into a
```
Comment (finp) [inline modules/ROOT/pages/_partials/generic-service-intros.adoc]: not what was agreed with PM/Eng, maybe put it somewhere else?

File: modules/ROOT/pages/digger/index.adoc [hunk 2]
```diff
@@ -3,10 +3,14 @@ include::{partialsdir}/attributes.adoc[]
 //':context:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
 :context: con_introduction-to-build
 
-[id='{context}_con_introduction-to-build']
-= About {digger-service}
+[[overview]]
+== Overview
```
Comment (finp) [inline modules/ROOT/pages/digger/index.adoc]: not the pattern we're using 
https://docs.google.com/presentation/d/1wxHPTo6mDRGQIsGAJBmJaagZIy2d4Chfq22Hqy5ztso/edit?usp=sharing

File: modules/ROOT/pages/digger/ref_terminology.adoc [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+[id='{context}_ref_terminology']
+
+= Reference material
+
+This section describes terminology that is associated with {digger-service}.
+
+CI/CD:: Continuous Integration / Continuous Development
+
+Jenkins:: A service used to instrument CI/CD solutions
```
Comment (finp) [inline modules/ROOT/pages/digger/ref_terminology.adoc]: Jenkins:: A tool that helps to automate  the software development process, with continuous integration and facilitating continuous delivery.

File: modules/ROOT/pages/_partials/generic-service-intros.adoc (hunk 1 above)
Comment (philbrookes) [reply modules/ROOT/pages/_partials/generic-service-intros.adoc]: Hey @finp where can I see what we did agree with them?

Comment (philbrookes) [reply modules/ROOT/pages/_partials/generic-service-intros.adoc]: Is that the original text?

File: modules/ROOT/pages/digger/ref_terminology.adoc (hunk 3 above)
Comment (mikenairn) [reply modules/ROOT/pages/digger/ref_terminology.adoc]: I'd agree this should probably be re-worded, @finp suggestions sounds good to me.

File: modules/ROOT/pages/digger/index.adoc (hunk 2 above)
Comment (philbrookes) [reply modules/ROOT/pages/digger/index.adoc]: hey @finp I copied this from the push index.adoc, that link appears to go to a slide in a slide show which reads "flexible customer content"? Could you let me know a good example which I could copy the pattern from?

Comment (philbrookes) [reply modules/ROOT/pages/digger/index.adoc]: also, I don't know if it's related but my index.adoc doesn't seem to be setting a title in the browser (neither does push which is what I copied from) can you let me know how I set a title?

Comment (finp) [reply modules/ROOT/pages/digger/index.adoc]: @philbrookes  I raised https://github.com/aerogear/mobile-docs/issues/73

will follow up with team on this naming, so ignore me ;)
... for now

Comment (philbrookes) [reply modules/ROOT/pages/digger/index.adoc]: I have figured out how to set the title now.

**Pull request merged**